README.md

```diff
@@ -1 +1,14 @@
-# Dx_BAF
\ No newline at end of file
+# Dx_BAF
+
+All script are made tested using Python 3.6.8
+
+## Installation
+To run the BAF scripts we need to create a virtual python environment
+
+### Make virtual python enviroment
+```bash
+python3 -m venv venv
```

Comment:
upgrade pip does not seem neccesary,

Additionally, I would say something like "in the main folder, use this command to make the virtual environment" or something similar. You can make a virtual environment named "venv" anywhere.

Reply:
Upgrade is recommended on our HPC (maybe even required), and does no harm.

Location of the venv folder is the main folder is not required (could be any location)